src/rust/ide/src/controller/graph/executed.rs

```diff
@@ -143,23 +143,16 @@ impl Handle {
         futures::stream::select_all(vec![value_stream,graph_stream,self_stream])
     }
 
-    /// Enter node by given ID.
+    /// Enter node by given node ID and method pointer.
     ///
     /// This will cause pushing a new stack frame to the execution context and changing the graph
     /// controller to point to a new definition.
-    ///
-    /// Fails if there's no information about target method pointer (e.g. because node value hasn't
-    /// been yet computed by the engine) or if method graph cannot be created (see
-    /// `graph_for_method` documentation).
-    pub async fn enter_node(&self, node:double_representation::node::Id) -> FallibleResult<()> {
-        debug!(self.logger, "Entering node {node}.");
-        let registry   = self.execution_ctx.computed_value_info_registry();
-        let node_info  = registry.get(&node).ok_or_else(|| NotEvaluatedYet(node))?;
-        let method_ptr = node_info.method_pointer.as_ref().ok_or_else(|| NoResolvedMethod(node))?;
-        let graph      = controller::Graph::new_method(&self.logger,&self.project,&method_ptr).await?;
-        let call       = model::execution_context::LocalCall {
+    pub async fn enter_method_pointer
```

Comment:
I'd like to keep log on the method that actually switches the graph.
This is a change of significant effects and should be somehow recorded.